integration/testrunner/grpc.go

```diff
@@ -308,28 +347,61 @@ func createConnection(ctx context.Context, t *testing.T, addr, idToken string) *
 // We specifically look up the log using the UUID specified in the request as we know the server will add that
 // as the resource name, and provides us a unique key to find logs with.
 func (g *GRPC) makeQueryForGRPCUnary(id string) *bigquery.Query {
-	// Cast to int64 because the result checker expects a number.
-	queryString := fmt.Sprintf("SELECT CAST(EXISTS (SELECT * FROM `%s.%s` WHERE jsonPayload.resource_name=?", g.ProjectID, g.DatasetQuery)
-	if g.RequireJustification {
-		// TODO(#265): For back-compatibility, we need to ensure justification in either metadata key.
-		queryString += ` AND jsonPayload.metadata.justification IS NOT NULL`
+	queryString := "SELECT "
+	queryString += fmt.Sprintf("%s as %s, ", "jsonPayload.method_name", "MethodName")
+	queryString += fmt.Sprintf("%s as %s, ", "jsonPayload.authentication_info.principal_email", "PrincipalEmail")
+	queryString += fmt.Sprintf("%s as %s, ", "jsonPayload.service_name", "ServiceName")
+	queryString += fmt.Sprintf("FROM `%s.%s` WHERE jsonPayload.resource_name='%s'", g.ProjectID, g.DatasetQuery, id)
```

Comment:
```suggestion
	queryString := fmt.Sprintf(`
		jsonPayload.method_name AS MethodName,
		jsonPayload.authentication_info.principal_email AS PrincipalEmail,
		jsonPayload.service_name AS ServiceName
	FROM `%s.%s`
	WHERE jsonPayload.resource_name = ?
	`, g.ProjectID, g.DatasetQuery)
	return makeQuery(*g.BigQueryClient, queryString, id)
```

We don't want to use string interpolation for the ID, since it's vulnerable to SQL injection.

Reply:
Fixed! Thank you for all these nice comments!